metacat-common-server/src/main/java/com/netflix/metacat/common/server/usermetadata/AuthorizationService.java

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.netflix.metacat.common.server.usermetadata;
+
+import com.netflix.metacat.common.QualifiedName;
+import com.netflix.metacat.common.server.connectors.ConnectorRequestContext;
+
+/**
+ * Authorization Service API.
+ *
+ * @author zhenl
+ * @since 1.2.0
+ */
+public interface AuthorizationService {
+    /**
+     * Config METACAT_CREATE.
+     */
+    String METACAT_CREATE = "create";
+    /**
+     * Config METACAT_DELETE.
+     */
+    String METACAT_DELETE = "delete";
+
+    /**
+     * Check db acl if this operation is permitted.
```

Comment:
Description should be any object referenced by the QualifiedName

Reply:
will do.